lib/neo4j/active_node/persistence.rb

```diff
@@ -48,13 +49,11 @@ def create_model(*)
       create_magic_properties
       set_timestamps
       create_magic_properties
-      properties = convert_properties_to :db, props
+      properties = self.class.declared_property_manager.convert_properties_to(self, :db, props)
```

Comment:
Perhaps make a instance method of `declared_property_manager` and have it call the class method to make this a bit simpler?